sites/hn.svelte.dev/vite.config.js

```diff
@@ -1,8 +1,19 @@
 import { sveltekit } from '@sveltejs/kit/vite';
+import browserslist from 'browserslist';
+import { browserslistToTargets } from 'lightningcss';
 
 /** @type {import('vite').UserConfig} */
 const config = {
-	plugins: [sveltekit()]
+	plugins: [sveltekit()],
+	css: {
```

Comment:
I'm kind of unsure about using lightingcss here actually. I think it's fine on svelte.dev/kit.svelte.dev, but it seems to complicate the setup and probably isn't ideal for a demo app. If we think it's better we should probably push to make it the default in SvelteKit or Vite instead

Reply:
Sure! Will remove